translation/transport.go

```diff
@@ -102,6 +102,18 @@ func getPartnerIDs(h http.Header) []string {
 	return partners
 }
 
+func getTraceParent(h http.Header) string {
+	tp := h.Get("traceparent")
+	tp = "traceparent: " + tp
+	return tp
+}
+
+func getTraceStatus(h http.Header) string {
+	ts := h.Get("tracestatus")
```

Comment:
If `ts` is `nil`, the header will still be sent, right?

Reply:
Yes, it will still send. I just realized this is probably not the desired behavior as it might trigger other systems to send traces since an empty traceparent will be present. I will update this to account for this case.